src/config-utils.coffee

```diff
@@ -0,0 +1,44 @@
+fs = require 'fs'
+yaml = require 'js-yaml'
+
+configUtils = {}
+
+configUtils.save = (args, path) ->
+  path = './dredd.yml' unless path?
+
+  args['blueprint'] = args['_'][0]
+  args['endpoint'] = args['_'][1]
+
+  for key, value of args
+    delete args[key] if key.length == 1
```

Comment:
I'd vote to use `options.coffee` file as source for _aliases_ here. It might resolve any possible problems with `argv`

Also - mutating args is probably not the best idea.